README.md

```diff
@@ -74,12 +64,11 @@ Workmanager.registerPeriodicTask(
 )
 ```
 
-You will need to provide a unique name; this comes in handy when you want to cancel this task later on.  
-The second parameter is the `String` that will be returned to your `callbackDispatcher` function.  
-You can use this `String` to identify which work needs to be done.  
+Each task must have a **unique name** ; this allows cancellation of a started task.  
+The second parameter is the `String` that will be sent to your `callbackDispatcher` function, indicating the task's *type*.  
 
 ## Customisation
```

Comment:
Maybe we should have the Customisation header under the Android usage header, so it's more clear this can only work for Android